src/game/Pet.cpp

```diff
@@ -932,6 +932,9 @@ void Pet::GivePetXP(uint32 xp)
     if (level >= maxlevel)
         return;
 
+    if (float rateXP = sWorld.getConfig(CONFIG_FLOAT_RATE_PET_XP_KILL))
```

Comment:
I did as you said, I just forgot to push the commit since I begang working on some allignment of the function (for code style)... Please don't think that I don't respect or listen to you, I really do.